docker-compose.yml

```diff
@@ -27,7 +27,9 @@ services:
   aws-cli:
     image: amazon/aws-cli
     env_file: .env
-    command: ["--endpoint-url", "$S3_ENDPOINT", "s3api", "create-bucket", "--bucket", "$S3_BUCKET"]
+    command: ["--endpoint-url", "http://minio:9000", "s3api", "create-bucket", "--bucket", "$S3_BUCKET"]
```

Comment:
Sorry for tagging along on your PR, @jasalisbury. I found this while reviewing this PR, but the change it totally unrelated to your changes.